THIRD_PARTY.md

```diff
@@ -0,0 +1,63 @@
+# Third party components
+
+This file contains information about third party components, required for
```

Comment:
Should we move third parties to a separate directory ? It would make sense in terms of licensing. This way we could internalize again zlib and spdlog easily if we want to

Reply:
This is a personal preference so feel free to ignore me; I like single-word names like `externals` whenever possible. If not possible, underscores over spaces.

Reply:
How about `thirdparty/libmve`?